Mathlib/GroupTheory/Commutator/Basic.lean

```diff
@@ -229,3 +209,6 @@ theorem mem_commutatorSet_iff : g ∈ commutatorSet G ↔ ∃ g₁ g₂ : G, ⁅
 
 theorem commutator_mem_commutatorSet : ⁅g₁, g₂⁆ ∈ commutatorSet G :=
   ⟨g₁, g₂, rfl⟩
+
+assert_not_exists Cardinal
+assert_not_exists Multiset
```

Comment:
Can you please put those at the top?

Reply:
We're actually already split in Mathlib as to whether these go at the top or bottom. I'm unsure myself which is better. At the top means they are closer to the import statements, at the bottom they are less clutter in the way of a normal reader who just wants to start reading declarations.

Reply:
Looking for consensus at https://leanprover.zulipchat.com/#narrow/stream/345428-mathlib-reviewers/topic/.60assert_not_exists.60.20at.20top.20or.20bottom.20of.20file.3F/near/459960523. I'll mark these as resolved in the meantime and we can cleanup later as required.